### PR TITLE
added web page browsing

### PR DESCRIPTION
**Pull request by AndyC2513**
pls approve ._.

### CODE REVIEW

File: backend/server.js
Comment (jeffreyzhai123) [inline backend/server.js:1]: hmmm very good

**Review by jeffreyzhai123 — APPROVED** [review]
lgtm

File: backend/server.js
Comment (joshhwuu) [reply backend/server.js:1]: wow yes.

**Review by joshhwuu — COMMENTED** [review]
so good.

**Review by joshhwuu — APPROVED** [review]
great job!

**Pull request merged**